common/scala/src/main/scala/whisk/core/entity/ActivationId.scala

```diff
@@ -88,13 +91,23 @@ protected[core] object ActivationId extends ArgNormalizer[ActivationId] {
         def read(value: JsValue) = Try {
             val JsString(s) = value
```

Comment:
Addressed in https://github.com/openwhisk/openwhisk/pull/918/commits/c057031eb1661743eb87539d424b97959544b102